adoc/Release-Notes.adoc

```diff
@@ -95,7 +164,6 @@ For information about deploying and administering {product}, see the product man
 ** loggregator: 104.4
 ** loggregator-agent: 3.2
 ** nats: 26
-** nfs-volume: 1.7.6
```

Comment:
This is the 1.4.0 section -- accidental removal or intentional? (Albeit the version number is awfully high compared to 1.4.1.)

Reply:
Removed from 1.4.0 for now until we sort out what the actual version was since we definitely didn't downgrade that component. Will add back in later with a correct number once found.

Reply:
https://github.com/SUSE/scf/blob/2.16.4/container-host-files/etc/scf/config/role-manifest.yml#L62 says we actually had nfs-volume-release 1.5.2 in CAP 1.4.0.